rotkehlchen/api/v1/schemas.py

```diff
@@ -620,7 +620,9 @@ class HistoryBaseEntrySchema(Schema):
         load_default=HistoryEventSubType.NONE,
         allow_none=True,
     )
-    counterparty = fields.String(required=False)
+    counterparty = fields.String(load_default=None)
```

Comment:
As discussed here: https://github.com/rotki/rotki/pull/5726/files#r1138462066

We need to figure out if this should be counterparty or protocol. Is the only reason it's counterparty since it can also be an exchange name for deposits/withdrawal EVM transactions?